From 7fb336c5cf412a5edc516c752476b4c3b0f618ac Mon Sep 17 00:00:00 2001
From: Tzafrir Cohen <tzafrir@debian.org>
Date: Fri, 31 Oct 2014 23:14:49 +0200
Subject: Reduce number of libraries linked with pjlib-util

The -util library hardly has any external library dependencies. Removing
the external libs.
---
 pjlib-util/build/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pjlib-util/build/Makefile b/pjlib-util/build/Makefile
index cb601cb..f89f62d 100644
--- a/pjlib-util/build/Makefile
+++ b/pjlib-util/build/Makefile
@@ -29,7 +29,7 @@ export _CFLAGS 	:= $(CC_CFLAGS) $(OS_CFLAGS) $(HOST_CFLAGS) $(M_CFLAGS) \
 export _CXXFLAGS:= $(_CFLAGS) $(CC_CXXFLAGS) $(OS_CXXFLAGS) $(M_CXXFLAGS) \
 		   $(HOST_CXXFLAGS) $(CXXFLAGS)
 export _LDFLAGS := $(CC_LDFLAGS) $(OS_LDFLAGS) $(M_LDFLAGS) $(HOST_LDFLAGS) \
-		   $(APP_LDFLAGS) $(LDFLAGS)
+		   $(APP_LDFLAGS)
 
 ###############################################################################
 # Defines for building PJLIB-UTIL library
